packages/formstr-sdk/src/formstr/formstr.ts

```diff
@@ -101,11 +102,11 @@ function convertV1Form(formSpec: V0FormSpec): V1FormSpec {
   return finalSchema;
 }
 
-export const getFormTemplate = async (npub: string) => {
+export const getFormTemplate = async (formId: string): Promise<V1FormSpec> => {
   const pool = new SimplePool();
   const filter = {
     kinds: [0],
-    authors: [npub],
+    authors: [formId],
```

Comment:
Authors are form id?

Can add a comment here as it does not look intuitive as to why the formid goes in author

Reply:
Sure, essentially formId is the npub, will add comment